internal/handlers/handlers.go

```diff
@@ -171,21 +169,19 @@ func (h *Handler) ProcessVerificationResult() fiber.Handler {
 		)
 		sigHeader := c.Get("Idenfy-Signature")
 		if len(sigHeader) < 1 {
+			h.logger.Error("Missing signature header", "headers", string(c.Request().Header.Header()))
```

Comment:
lower case the error messages except for nouns

Reply:
I will handle all log messages code-wide in separate PR. need to improve:
- properly structured logs
- revise log levels
- revise messages case
- consistency

Reply:
Opened an issue here: https://github.com/threefoldtech/tf-kyc-verifier/issues/20